src/components/Card/Card.css

```diff
@@ -94,3 +94,24 @@
 .fa.fa-heart {
   color: red;
 }
+.quantity-container {
+  display: flex;
+  gap: 10px;
+  /* justify-content: baseline;
```

Comment:
try removing commented code.

Reply:
Will remove the commented coded thank you ayushi